bundles/binding/org.openhab.binding.zwave/database/linear/wadwaz-1.xml

```diff
@@ -22,41 +22,8 @@
 			<id>0xFF</id>
 		</Class>
 	</CommandClasses>
-	<Configuration>
-		<Parameter>
-			<Index>1</Index>
-			<Type>short</Type>
-			<Default>0</Default>
-			<Size>2</Size>
-			<Label Lang="en">IN1 Alarm Cancellation Delay alarm Cancellation
-			</Label>
-			<Help Lang="en">Additional delay after an alarm from input IN1 has
-				ceased. The parameter allows you to specify additional time, after
-				which the input no. 1 alarm is cancelled once its violation has
-				ceased.
-			</Help>
-		</Parameter>
-		<Parameter>
-			<Index>2</Index>
-			<Type>byte</Type>
-			<Default>1</Default>
-			<Size>1</Size>
-			<Label Lang="en">Status change signaled by LEDs change signaled by
-			</Label>
-		</Parameter>
-	</Configuration>
 	<Associations>
 		<Group>
-			<Index>1</Index>
-			<Maximum>5</Maximum>
-			<Label lang="en">Input IN1</Label>
-		</Group>
-		<Group>
-			<Index>2</Index>
-			<Maximum>5</Maximum>
-			<Label lang="en">Tamper Button</Label>
-		</Group>
-		<Group>
 			<Index>3</Index>
```

Comment:
What software are you using to interface with Git? I would recommend SourceTree.

If using SourceTree, it will show files that have changed with a yellow circle, and ones that are ‘new’ with a blue circle with a question mark in it.  You will get new files pop up from time to time - temp files that the compiler makes or whatever… I just ignore these… (i.e. I don’t do anything). You can set up a gitignore file to add these files (or folders) into and then they are ignored - I suspect that this might be what ‘ignore’ does, but I’ve not tried it (I normally edit the gitignore file by hand).

If files actually ‘change’ then you need to ‘reset’ them, which changes them back to the state they were at the last commit.